code/modules/vehicles/powerloader.dm

```diff
@@ -17,16 +17,32 @@
 	var/open_state = "powerloader_open"
 	var/overlay_state = "powerloader_overlay"
 	var/wreckage = /obj/structure/powerloader_wreckage
+	var/obj/item/powerloader_clamp/PC_left
+	var/obj/item/powerloader_clamp/PC_right
+
+//--------------------GENERAL PROCS-----------------
 
 /obj/vehicle/powerloader/Initialize()
 	cell = new /obj/item/cell/apc
-	for(var/i = 1, i <= 2, i++)
-		var/obj/item/powerloader_clamp/PC = new(src)
-		PC.linked_powerloader = src
+	PC_left = new(src)
+	PC_left.name = "\improper Power Loader Left Hydraulic Claw"
+	PC_left.linked_powerloader = src
+	PC_right = new(src)
+	PC_right.name = "\improper Power Loader Right Hydraulic Claw"
```

Comment:
should be fairly simple to just swap the claw dmis for the existing claw and apply the icon state to the right claw, so they have congruent and correct sprites

Reply:
hm, Can be done, I suppose